lib/src/theme/tokens/radius_token.dart

```diff
@@ -58,27 +50,3 @@ class RadiusRef extends Radius with TokenRef<RadiusToken, Radius> {
   @override
   int get hashCode => token.hashCode;
 }
-
-// // Helper class to wrap functions that can return
-// a Radius token that resmebles the WithSpaceToken
-@immutable
-class UtilityWithRadiusTokens<T> {
```

Comment:
Are the other utilities still within Mix? I would like to maintain this here, to help compose utilities.

Reply:
Leo, are you sure about this? This class wasn't instantiated anywhere in the code.